### DISCUSSION TITLE
1.1.19

### DISCUSSION THREAD

**Discussion by wstyres** [wstyres/Zebra#1846] [Releases]
Fixed in this version:
- Workaround for not being able to send support emails from within Zebra
- Temporarily removed the source clipboard feature

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.19'>1.1.19</a>.</em>

**Comment by 00p513-dev** [comment]
Are you going to upload the deb?

**Reply by wstyres** [reply]
Sorry about that, just posted it on the release page. For future reference, debs can always be obtained from our APT repo directly or from the repository at https://github.com/zbrateam/Safari/tree/main/repo/pool.

**Reply by 00p513-dev** [reply]
Thank you :)
